src/components/Nav/useNav.ts

```diff
@@ -21,7 +21,7 @@ import {
 import { PiFlask, PiUsersFourLight } from "react-icons/pi"
 import { useColorMode } from "@chakra-ui/react"
```

Comment:
I assume we're leaving this for now to allow Chakra to handle the color-mode toggling still?